ninja/errors.py

```diff
@@ -43,6 +43,12 @@ def __init__(self, status_code: int, message: str) -> None:
         super().__init__(message)
 
 
+class MethodNotAllowed(Exception):
```

Comment:
Hello, I noticed that this new class was not declared in the `__all__` of this file. It must be declared so that it is available to be imported into other modules.

Reply:
Hello! Thanks for the review. I fixed it.